views.py

```diff
@@ -166,9 +166,9 @@ def page_detail(request, page_id, book_id=None):
     # If the page numbers don't match the list indices, search for the correct page
     if(book_json['relations']['hasPart'][pagenum - 1]['pid'] != page_pid):
         for page in book_json['relations']['hasPart']:
-            if(int(page['rel_has_pagination_ssim']) == (pagenum-1)):
+            if(int(page['rel_has_pagination_ssim'][0]) == (pagenum-1)):
```

Comment:
I would actually suggest using int(page['order']) here. See https://repository.library.brown.edu/api/pub/items/bdr%3A385513/ - in the hasPart section, each item has an 'order' field, and it's not an array.